CONTRIBUTING.md

```diff
@@ -96,6 +122,11 @@ Plus its just another thing to have to keep track of.
   - Use "refine.bio", NOT "refinebio"
   - Use `.Rmd`,  NOT "Rmd" or ".Rmd"
   - Use "tidyverse", not "Tidyverse"
+  - Use "TSV" vs tsv or `tsv` or .tsv
+  - Use "PNG" vs png or `png` or .png (and etc.)
```

Comment:
I heard you like consistent style, so I made a consistent style for your consistent style guide
```suggestion
  - Use "tidyverse", NOT "Tidyverse"
  - Use "TSV",  NOT tsv or `tsv` or .tsv
  - Use "PNG", NOT png or `png` or .png (and etc.)
```